src/calculateRentalCost.js

```diff
@@ -28,7 +28,25 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const dailyValue = 40;
+  const weekSale = 50;
+  const threeDaysSale = 20;
+  const week = 7;
+  const weekend = 3;
+
+  const weekendRent = days * dailyValue - threeDaysSale;
+  const weekRent = days * dailyValue - weekSale;
+  const dayRent = days * dailyValue;
+
+  if (days >= week) {
+    return weekRent;
+  }
+
+  if (days >= weekend && days < week) {
+    return weekendRent;
+  }
```

Comment:
```suggestion
  if (days >= weekend) {
    return dayRent - threeDaysSale;
  }
```
You have already checked it